### DISCUSSION TITLE
Unable to find template WebProfilerBundle:Profiler:layout.html.twig

### DISCUSSION THREAD

**Discussion by MounicaAruna** [pimcore/pimcore#9401] [Q&A / Support]
Hi,I'm getting the below error while hitting http://hostname/projectfolder/public/

**Unable to find template "WebProfilerBundle:Profiler:layout.html.twig" (looked into: /var/www/html/projectfolder/templates, /var/www/html/projectfolder/vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form) in @PimcoreCore/Profiler/data_collector.html.twig at line 1.**

1. Need help to find the solution. I'm new to pimcore.
2. Few weeks back we setup the pimcore skeleton code. It's working fine.At the time pimcore using symfony 4.4 version.
3. But now not working. Pimcore using symfony 5.3.

How to rectify this problem?

**Comment by dvesh3** [comment]
fixed in https://github.com/pimcore/pimcore/pull/9400/files

**Reply by MounicaAruna** [reply]
Hi @dvesh3 ,unable to find the following file bundles/CoreBundle/Resources/views/Profiler/data_collector.html.twig.
please share the full path of pimcore directory.

**Reply by dvesh3** [reply]
@MounicaAruna you are right, we need to backport the fix on 6.9 branch.

**Reply by dvesh3** [reply]
fixed here https://github.com/pimcore/pimcore/commit/9433d1dd67e999c9abe011ea5eb603497e41898d
